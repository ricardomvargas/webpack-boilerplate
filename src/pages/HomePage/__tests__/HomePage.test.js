test('HomePage dummy teste', () => {
    return undefined;
});
