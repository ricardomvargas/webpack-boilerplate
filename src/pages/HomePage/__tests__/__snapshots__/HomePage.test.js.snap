// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if match with snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="topbar"
      >
        <ul
          class="simple-menu"
        >
          <li>
            <a
              href="/"
            >
              Home
            </a>
          </li>
          <li>
            <a
              href="/form-sample"
            >
              Form Sample
            </a>
          </li>
        </ul>
      </nav>
      <article
        class="home-page"
      >
        <h1>
          Test Home Page
        </h1>
        <p
          class="project-intro"
        >
          Base web project to create applications with front end applications.
        </p>
        <section
          class="tools-block"
        >
          <a
            href="https://webpack.js.org/"
            target="_blank"
            title="Webpack"
          >
            <img
              alt="webpack"
              src="/assets/images/webpack-icon.svg"
            />
          </a>
          <a
            href="https://reactjs.org/"
            target="_blank"
            title="ReactJS"
          >
            <img
              alt="reactjs"
              src="/assets/images/react-icon.svg"
            />
          </a>
          <a
            href="https://reactrouter.com/"
            target="_blank"
            title="React Router"
          >
            <img
              alt="react-route"
              src="/assets/images/react-router-icon.svg"
            />
          </a>
          <a
            href="https://www.typescriptlang.org/"
            target="_blank"
            title="Typescript"
          >
            <img
              alt="typescript"
              src="/assets/images/typescript-icon.svg"
            />
          </a>
          <a
            href="https://jestjs.io/"
            target="_blank"
            title="Jest"
          >
            <img
              alt="jest"
              src="/assets/images/jest-icon.svg"
            />
          </a>
          <a
            href="https://testing-library.com/"
            target="_blank"
            title="Testing Library"
          >
            <img
              alt="testing-library"
              src="/assets/images/testing-library-icon.svg"
            />
          </a>
        </section>
        <section>
          <h2>
            Getting Started
          </h2>
          <ul>
            <li>
              Installing deppedencies: npm install
            </li>
            <li>
              Executing the project: npm run dev (it runs on port 3000)
            </li>
            <li>
              Executing tests: npm run test
            </li>
          </ul>
          <h2>
            What's next?
          </h2>
          <ul>
            <li>
              Layout design
            </li>
            <li>
              Project description and guideline
            </li>
            <li>
              More components samples
            </li>
            <li>
              Test samples and guidelines
            </li>
          </ul>
        </section>
      </article>
    </div>
  </body>,
  "container": <div>
    <nav
      class="topbar"
    >
      <ul
        class="simple-menu"
      >
        <li>
          <a
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            href="/form-sample"
          >
            Form Sample
          </a>
        </li>
      </ul>
    </nav>
    <article
      class="home-page"
    >
      <h1>
        Test Home Page
      </h1>
      <p
        class="project-intro"
      >
        Base web project to create applications with front end applications.
      </p>
      <section
        class="tools-block"
      >
        <a
          href="https://webpack.js.org/"
          target="_blank"
          title="Webpack"
        >
          <img
            alt="webpack"
            src="/assets/images/webpack-icon.svg"
          />
        </a>
        <a
          href="https://reactjs.org/"
          target="_blank"
          title="ReactJS"
        >
          <img
            alt="reactjs"
            src="/assets/images/react-icon.svg"
          />
        </a>
        <a
          href="https://reactrouter.com/"
          target="_blank"
          title="React Router"
        >
          <img
            alt="react-route"
            src="/assets/images/react-router-icon.svg"
          />
        </a>
        <a
          href="https://www.typescriptlang.org/"
          target="_blank"
          title="Typescript"
        >
          <img
            alt="typescript"
            src="/assets/images/typescript-icon.svg"
          />
        </a>
        <a
          href="https://jestjs.io/"
          target="_blank"
          title="Jest"
        >
          <img
            alt="jest"
            src="/assets/images/jest-icon.svg"
          />
        </a>
        <a
          href="https://testing-library.com/"
          target="_blank"
          title="Testing Library"
        >
          <img
            alt="testing-library"
            src="/assets/images/testing-library-icon.svg"
          />
        </a>
      </section>
      <section>
        <h2>
          Getting Started
        </h2>
        <ul>
          <li>
            Installing deppedencies: npm install
          </li>
          <li>
            Executing the project: npm run dev (it runs on port 3000)
          </li>
          <li>
            Executing tests: npm run test
          </li>
        </ul>
        <h2>
          What's next?
        </h2>
        <ul>
          <li>
            Layout design
          </li>
          <li>
            Project description and guideline
          </li>
          <li>
            More components samples
          </li>
          <li>
            Test samples and guidelines
          </li>
        </ul>
      </section>
    </article>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
